sources/XXFILL

```diff
@@ -572,35 +588,35 @@ Copyright (c) 1985-1990, 1993, 2021 by Venue & Xerox Corporation.
 
 (POLYSHADE.SCAN.IP
   [LAMBDA (STREAM POINTS TEXTURE OPERATION WINDNUMBER SCANDIRECTION)
+                                                             (* ; "Edited  2-May-2023 09:12 by lmm")
                                                              (* ; "Edited  1-Feb-89 18:53 by FS")
 
     (* ;; "Convert micas to device units, and transpose, tell Interpress to take dev units back to micas, convert knot list into internal data structures")
 
     (LET (ILIST LLIST (MicasToDev 0.1181102)
                 XTOX XTOY)                                   (* ; "hack until can change IP")
                                                              (* ; "MicasToDev 300dpi / 2540micaspi")
-         (IF (EQ SCANDIRECTION 'X)
-             THEN (SETQ XTOX MicasToDev)                 (* ; "just scale")
-                   (SETQ XTOY 0)
-           ELSE (SETQ XTOX 0)                            (* ; "transpose & scale")
-                 (SETQ XTOY MicasToDev))
-         [IF (NUMBERP (CAAR POINTS))
-             THEN (SETQ POINTS (IMLTLIST POINTS XTOX XTOY 0 XTOY XTOX 0))
-           ELSE (SETQ POINTS (for I in POINTS
-                                    collect (IMLTLIST I XTOX XTOY 0 XTOY XTOX 0]
-         (APPENDOP.IP STREAM DOSAVESIMPLEBODY)
-         (APPENDOP.IP STREAM {)
+         (if (EQ SCANDIRECTION 'X)
+             then (SETQ XTOX MicasToDev)                     (* ; "just scale")
+                  (SETQ XTOY 0)
+           else (SETQ XTOX 0)                                (* ; "transpose & scale")
+                (SETQ XTOY MicasToDev))
+         [if (NUMBERP (CAAR POINTS))
+             then (SETQ POINTS (IMLTLIST POINTS XTOX XTOY 0 XTOY XTOX 0))
+           else (SETQ POINTS (for I in POINTS collect (IMLTLIST I XTOX XTOY 0 XTOY XTOX 0]
+         (APPENDOP.IP STREAM (\IPC DOSAVESIMPLEBODY))
+         (APPENDOP.IP STREAM (\IPC {))
          (SCALE.IP STREAM 8.466666)                          (* ; "2540micaspi / 300dpi")
```

Comment:
Feels like this should be a constant we have...

Reply:
I think I'll try a different approach